<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>429</width>
      <height>573</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_del_all">
       <property name="text">
        <string>                   ви точно хочете все видалити ?</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="Enter_label">
         <property name="text">
          <string>уведіть назву демонстрації</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name_demonstration"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="enter_event">
         <property name="text">
          <string>уведіть назву події</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name_event"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="enter_probality">
         <property name="text">
          <string>уведіть ймовірність події</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="size_probality"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="DELL_event">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="back_from_dell_event">
         <property name="text">
          <string>відміна</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="ADD_event">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="back_from_add_event">
         <property name="text">
          <string>відміна</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="DELL_ALL_event">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="back_from_dell_all">
         <property name="text">
          <string>відміна</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="create_demonstration">
       <property name="text">
        <string>створити нову демонстрацію</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edit_demonstration">
       <property name="text">
        <string>редагувати демонстрацію</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_demonstration">
       <property name="text">
        <string>видалити демонстрацію</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_demonstration">
       <property name="text">
        <string>запустити демонстрацію</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_event">
       <property name="text">
        <string>видалити подію</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_event">
       <property name="text">
        <string>додати подію</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_all_event">
       <property name="text">
        <string>видалити всі події</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="read_event">
       <property name="text">
        <string>переглянути всі події даної демонстрації</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="return_to_menu">
       <property name="text">
        <string>повернутися на головну</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="num_experement">
         <property name="text">
          <string>уведіть кількість випробувань</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="enter_num_experement"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="edit_num_experement">
       <property name="text">
        <string>змінити кількість випобувань</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>261</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>321</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>Вивід</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="title">
     <string>статистиика</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>240</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>уведіть назву події</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name_event_for_statistics"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>уведіть назву демонстрації</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name_demomstration_for_statistic_"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="read_statistics">
       <property name="text">
        <string>перглянути статистику події</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>620</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>закрити програму</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>635</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
